jekyll/_cci2/api_v2_introduction.md

```diff
@@ -0,0 +1,226 @@
+
+Many users are already using an early version of pipelines, even before CircleCI exposed them through the CircleCI API and web UI. If you have already used the latest features in CircleCI version 2.1 configuration and/or orbs, then you already have pipelines enabled for your project (all projects added after September 1, 2018 already have pipelines enabled by default). If, however, your project does not yet have pipelines enabled, go to **Advanced Settings** in your project's **Settings** page. If you do not see the **Enable Pipelines** option on your project's settings page, this means your project already has pipelines enabled.
+
+#### Is a Pipeline the Same Thing as a Build?
+
+In previous CircleCI releases, the term "build" was used to describe a pipeline. Unfortunately, this terminology proved problematic because there was an incongruity between how "build" and "job" were defined in the API and UI. For example, what was referred to as a "build" in past releases is now referred to as a "job" in the current release. To address this ambiguity, CircleCI updated the UI to reflect this change in nomenclature, but there are still instances of "build" in our API.
+
+More importantly, CircleCI found that more teams are using CircleCI for a much broader array of automated processes than the word "build" implies. “Pipelines” more accurately reflects the broad applications of our platform, therefore, "pipelines" are more appropriate for the set of stages CircleCI performs in processing your automation instructions against changes in your code. 
+
+#### Will I Have to Use Pipelines?
+
+You are not required to use pipelines, but note that pipelines will eventually be enabled for all projects. This will allow CircleCI to provide more powerful configuration and automation, as well as better optimization of your resources and more reliable performance. The overarching goal of this approach is for the transition to pipelines to be seamless and have no disruptions on any of your existing workflows.
+
+### Pipeline Parameters
+
+Pipeline parameters are new with API v2. To use pipeline parameters you must use configuration version 2.1 or higher.
+
+#### Declaring and Using Pipeline Parameters in a Configuration
+
+Pipeline parameters are declared using a parameters stanza in the top level keys of your `.circleci/config.yml` file. You may then reference the value of the parameter as a config variable in the scope `pipeline.parameters`.
+
+The example belows shows a configuration with two pipeline parameters, `image-tag` and `workingdir` both used on the subsequent config stanzas:
+
+```yaml
+version: 2.1
+
+parameters:
+  image-tag:
+    type: string
+    default: "latest"
+  workingdir:
+    type: string
+    default: "~/main"
+
+jobs:
+  build:
+    docker:
+      - image: circleci/node:<< pipeline.parameters.image-tag >>
+    environment:
+      IMAGETAG: << pipeline.parameters.image-tag >>
```

Comment:
This is potentially confusing -- recommend instead you reference the pipeline parameter directly in the step below.

Reply:
Fixed